test/views/SubmitFormViewSpec.scala

```diff
@@ -174,5 +180,26 @@ class SubmitFormViewSpec extends ViewBaseSpec {
         elementText("label[for=box1]") shouldBe "Swm Blwch 1 TAW a godwyd gennych ar werthiannau a chyflenwadau eraill"
       }
     }
+
+    "page has errors" should {
+
+      lazy val view = submitFormView(
+        periodKey,
+        Some("ABC Studios"),
+        flatRateScheme = true,
+        obligation,
+        SubmitVatReturnForm().nineBoxForm.bind(Map("" -> "")),
+        isAgent = false
+      )(fakeRequest, messages, mockAppConfig, user)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct page title 'Error: Honesty declaration'" in {
```

Comment:
The title in the description here isn't consistent with what is being returned in the test. Rather than update it thought I think it's best to just remove mention of the value from the description.